src/MDMAnimationTraits.m

```diff
@@ -0,0 +1,23 @@
+/*
+ Copyright 2017-present The Material Motion Authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDMAnimationTraits.h"
+
+const MDMAnimationTraits MDMAnimationTraitsSystemModalMovement = {
+  .duration = 0.500, .timingCurve = {
+    .type = MDMTimingCurveTypeSpring, .data = { 3, 1000, 500, 0 }
```

Comment:
Nit/Follow-up: Naming the data?

Reply:
I'm not sure what or how we would name this - did you have something in mind?

Reply:
I don't. They just look like a series of NSInteger(uint32_t, int16_t, double?) values. I know they make a timing curve, but looking at the code I have no idea what they're doing.

Reply:
Ah, I had assumed you meant naming the `data` parameter. I'll give these values const names in the scope here.